src/main/java/com/statistic/deputies/controller/DeputyController.java

```diff
@@ -34,4 +38,13 @@
     public List<Deputat> deputiesWithAwards() {
         return deputyService.getDeputiesWithAwards();
     }
+
+    @GetMapping("/getAllDeputiesGroupByParty")
```

Comment:
@kenu21 немає сенсу дублювати deputy адже в тебе ця приставка вже є в контроллері
```suggestion
    @GetMapping("/getGroupedByParty")
```

Приставку all також можна прибрати, адже в майбутньому ти можеш додати деякі фільтри до цього запиту

Reply:
ОК.